src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.DataConverter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataConverterImpl implements DataConverter {
+    public static final String SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> records) {
+        return records.stream()
+                .skip(1)
```

Comment:
Skipping the first line is a detail specific to the current file format. Consider making your service more flexible by not assuming the presence of a header or by providing a way to handle files without headers.